backend/src/main/java/net/es/oscars/sb/nso/NsoLspStateSyncer.java

```diff
@@ -221,16 +227,25 @@ public boolean sync(String path, boolean dryRun) throws NsoStateSyncerException
                 // @TODO This for loop could be parallelized.
                 // @TODO The redeploys MIGHT need to be ordered in a certain way if there are resource dependencies
                 for (NsoStateWrapper<NsoLSP> wrapper : toRedeploy) {
+                    NsoLSP lsp = wrapper.getInstance();
+                    if (!this.isOscarsManaged(lsp)) {
+                        log.error("attempted to redeploy unmanaged LSP " + lsp.getName() + "/" + lsp.getDevice());
```

Comment:
either this should be a warning, or an exception should be thrown if truly an error.